src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -1,14 +1,12 @@
 // @debug $about-modal-box; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$about-modal-box} {
+:root {
   // Component variables
   --#{$about-modal-box}--BackgroundImage: none;
-  --#{$about-modal-box}--BackgroundColor: var(--#{$pf-global}--palette--black-1000); // Modal uses a non-standard background color
+  --#{$about-modal-box}--BackgroundColor: var(--pf-t--global--background--color--floating--default);
   --#{$about-modal-box}--BackgroundPosition: bottom right;
-  --#{$about-modal-box}--BackgroundSize--min-width: 200px;
-  --#{$about-modal-box}--BackgroundSize--width: 60%;
-  --#{$about-modal-box}--BackgroundSize--max-width: 600px;
-  --#{$about-modal-box}--BackgroundSize: clamp(var(--#{$about-modal-box}--BackgroundSize--min-width), var(--#{$about-modal-box}--BackgroundSize--width), var(--#{$about-modal-box}--BackgroundSize--max-width));
+  --#{$about-modal-box}--BackgroundSize--width: 100%;
+  --#{$about-modal-box}--BackgroundSize: contain;
```

Comment:
We should nail down what the intent is with the background image - @lboehling @andrew-ronaldson do we have a design guideline for what kind of image to use here? And is there any reason  If you put a wide image in now, with `contain`, it will fill the whole box - https://codepen.io/mcoker/pen/PoVpVqQ

Doesn't need to hold up this PR, just something to consider. Also the about modal and background image component CSS is basically the same. The login page uses the background image component - seems like the about modal could, too, and we could just remove the background image specific styles from the about modal?

Reply:
I took out the sizing because it's fairly specific to the image anyway and this one needs to span the full height or it will look odd. And there will need to be guidelines on making the image since it needs to handle the switch to dark theme.
I do think we should probably just use the background component here?